README.md

```diff
@@ -224,7 +224,7 @@ Defaults for AEMM configuration are sourced throughout code. Examples below:
 * **Metadata mock responses**
   * [aemm-metadata-default-values.json](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/pkg/config/defaults/aemm-metadata-default-values.json)
 * **Commands**
-  * [events](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/pkg/cmd/scheduledevents/scheduledevents.go#L72) 
+  * [events](https://github.com/aws/amazon-ec2-metadata-mock/blob/master/pkg/cmd/events/events.go#L72) 
```

Comment:
Thinking big picture here, maybe we should consider tagging ReadMe changes as [upcoming] or with the next version number or something when they refer to behavior that depends on a future release. And the old readme line would be tagged with the current version number instead of totally removed. 

Just trying to brainstorm a good way to avoid having someone try out an example from the readme that doesn't work yet. If you think there's value here maybe we can discuss with the team

Reply:
These are good points. I think the tough part is figuring out if ReadMe should reflect *latest code* vs. *latest release*. 

As it stands, this ReadMe reflects the functionality of the latest code, but I get your point if someone followed installation instructions, downloaded latest release, and tried to follow the rest of the ReadMe it wouldn't necessarily be accurate. 

Agree we should discuss within the team and do some more deep dive as to how other projects might be handling it